src/bowlingGame.js

```diff
@@ -0,0 +1,104 @@
+function BowlingGame() {
```

Comment:
Think about separating out the responsibilities of the bowling game, ex. frame, scorecard